web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1767,23 +1763,83 @@ div#edit-individual-profile-consent-job-update-wrapper, div#edit-candidate-profi
   margin-top: -19px;
   left: 10px;
 }
-.collapsible-filter-block.collapsible-filter-block-processed.collapsible-filter-block-expanded > h2 {
-  margin-bottom: 1em;
-}
-.collapsible-filter-block.collapsible-filter-block-processed.collapsible-filter-block-expanded {
-  padding-bottom: 1em;
-}
+
 .collapsible-filter-block.collapsible-filter-block-expanded > h2:before {
   content: url(../assets/icon/minus_white.png);
 }
 
+.col-xs-12.col-md-6.layout__region.layout__region--first {
```

Comment:
There's a better class for this I'm sure. Maybe add pl-24 and pr-24 to the element.